README.md

```diff
@@ -4,6 +4,44 @@ Shimmer is a collection of Rails extensions that bring advanced UI features into
 
 ## Features
 
+### Rubocop Base Configuration
+
+Maintaining multiple *Rubocop* configurations, in multiple projects, is a bit of a nightmare. We want to unify the way we work on projects at *nerdgeschoss* and since every project will pull *Shimmer* as one of its base dependency, it's the perfect place to maintain a shared *Rubocop* configuration.
```

Comment:
New version:

### Rubocop Base Configuration

*Shimmer* offers an opiniated *Rubocop* base configuration. This configuration inherits itself from *StandardRB* and aim at remaining as close to it as possible. Why not only use *StandardRB*, since it is so fast and prevent bikeshedding? Well, sadly, it does not solve all problems and using *Rubocop* still integrates a lot easier in most toolsets. However, the idea is to still prevent bikeshedding our *Rubocop* configuration by making sure that every exception to what's configured in *StandardRB* is justified (with a comment over its configuration block in `./config/rubocop_base.yml`), reviewed, debated, and agreed upon before being merged.